jekyll/_cci2/pipelines.adoc

```diff
@@ -65,14 +79,16 @@ To add or edit a pipeline, follow these steps:
 . Select btn:[Add Pipeline], or, if you wish to edit an existing pipeline select the pencil icon on the right.
 . Complete the form fields and options:
 ** Give your pipeline a descriptive name, for example `Run tests and deploy`.
-** Under Integration select the platform that matches where your project is set up (for example, GitHub App).  If you integrate with  xref:github-integration#[CircleCI's GitHub OAuth App]) you can select GitHub App here to install the App into your org and access xref:github-integration#using-github-app-functionality[improved security and new features].
+** Under Integration select the platform that matches where your project is set up (for example, GitHub App).
++
+NOTE: If you integrate with xref:github-integration#[CircleCI's GitHub OAuth App] you can select GitHub App here to install the App into your org and access xref:github-integration#using-github-app-functionality[improved security and new features].
```

Comment:
I think we could add here that this is a one-time step that must be done by an organization or repo admin